src/seedu/addressbook/storage/StorageFile.java

```diff
@@ -79,32 +84,47 @@ public StorageFile(String filePath) throws InvalidStorageFilePathException {
     }
 
     /**
-     * Returns true if the given path is acceptable as a storage file.
-     * The file path is considered acceptable if it ends with '.xml'
+     * Returns true if the given path is acceptable as a storage file. The file
+     * path is considered acceptable if it ends with '.xml'
      */
     private static boolean isValidPath(Path filePath) {
         return filePath.toString().endsWith(".xml");
     }
 
     /**
-     * Saves all data to this storage file.
+     * Checks if the save file still exists
      *
-     * @throws StorageOperationException if there were errors converting and/or storing data to file.
+     * @throws StorageOperationException
+     *             if there were errors converting and/or storing data to file.
      */
-    public void save(AddressBook addressBook) throws StorageOperationException {
+    public void checkSaveFile(AddressBook addressBook) throws StorageOperationException {
```

Comment:
Based on the implementation for this method, does the method name `checkSaveFile(...)` conveying what the method does?
In other words, if it is checking for save file, why does it write to file?
Could a better name say `checkAndSaveToFile(...)` be more readable?